### ISSUE TITLE
Flash Attention in Gemma2 will never use the sliding window attention mask

### ISSUE THREAD

**Issue by abheesht17** [keras-team/keras-hub#2186]
We compute the sliding window attention mask here: https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/gemma/gemma_attention.py#L175-L179?

Shouldn't we do it before the Flash Attention branch: https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/gemma/gemma_attention.py#L136?

**Comment by abheesht17** [comment]
Resolved in https://github.com/keras-team/keras-hub/pull/2187
